tests/validation/model_workloads.yaml

```
@@ -303,3 +303,5 @@ models:
     workloads: [stsbenchmark_encoder_embedding_parity]
   z-image-turbo:
     workloads: [dpg_bench_diffusion_image]
+    s1-mini-fp16:
+    workloads: [mmlu_five_shot_mcq]
```

YAMLlint (1.37.1)

[error] duplication of key "workloads" in mapping

(key-duplicates)